sympy/physics/vector/dyadic.py

```diff
@@ -55,6 +56,12 @@ def __init__(self, inlist):
                 i -= 1
             i += 1
 
+    @property
+    def func(self):
+        return Dyadic
+
+    is_number = False
+
```

Comment:
What is the purpose of `func()` and `is_number`? Why do we need to add these?

Reply:
.evalf() looks for func() and is_number when called, these two are needed for evalf to work.